File: trunk/server/net/Protocols/BaseActionRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// base action 协议:24
    /// </summary>
	[Proto(value=24,description="base action")]
	[ProtoContract]
	public class BaseActionRequest
	{
        /// <summary>
        ///  roomId
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;
        /// <summary>
        ///  target
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string targetId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/BaseActionRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = 24, description = "base action")
public class BaseActionRequest
{
	@Protobuf(description = "roomId")
	private String roomId;
	@Protobuf(description = "target")
	private String targetId;
}
